tasks/GPSDTask.cpp

```diff
@@ -1,4 +1,14 @@
-#include "GPSDTask.hpp"
+
+// workaround for name clash in 
+// usr/include/mach/policy.h and gpsd/3.15/include/gps.h
```

Comment:
I basically rename policy_t to mach_policy_t insight mach/policy.h using the pre compiler.  After that I can undef policy_t which would not be possible if mach/policy.h would have used it for its typedef. Now, the policy_t is free to be used by gps.h for its struct.




and libgpsmm.h is free to use it.